scripts/scriptutils/helpers.py

```diff
@@ -1,5 +1,5 @@
 import math
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
Is `iterable` a type options? If so, we can get backward compatibility through `from __future__ import annotations`

Reply:
So using __future__ does allow collections to be backwards compatible, but v3.8 didn't yet have .removesuffix() for strings which is used through part_retrieve.py. 

Added some handling to this to get around this issue, but 2 tests still fail in py v3.8 as filecmp asserts deem integrated package and collated file to be different   in test_collation  and  test_all_import_types.

Reply:
Those issues are likely due to the update in rdflib that causes it to no longer have a spurious newline, which I fixed in #311. If you update your branch, it should resolve those issues.